yarn.lock

```diff
@@ -9166,6 +9153,19 @@ __metadata:
   languageName: node
   linkType: hard
 
+"old-hd-keyring@npm:@metamask/eth-hd-keyring@^4.0.1":
+  version: 4.0.2
+  resolution: "@metamask/eth-hd-keyring@npm:4.0.2"
```

Comment:
We are not longer testing the `4.0.1`, probably because of the `^` now.

What was the `yarn lint` issue you got? Was it caused by syncpack?

Reply:
As discussed internally, that should be ok to use the `4.0.2` for our test. 👍